ytypes/schema_tests/set_test.go

```diff
@@ -50,9 +50,199 @@ func TestSet(t *testing.T) {
 		inPath           *gpb.Path
 		inValue          *gpb.TypedValue
 		inOpts           []ytypes.SetNodeOpt
+		inGetOpts        []ytypes.GetNodeOpt
 		wantErrSubstring string
 		wantNode         *ytypes.TreeNode
 	}{{
+		desc:     "set-on-union-with-invalid-string-but-valid-enum",
```

Comment:
nit: why "-" separated and not " "

Reply:
go/gotip/episodes/117: "Avoid non-alphabetic characters"